integration-tests/src/test_loop/tests/state_sync.rs

```diff
@@ -74,11 +74,12 @@ fn setup_initial_blockchain(
     num_shards: usize,
     generate_shard_accounts: bool,
     chunks_produced: HashMap<ShardId, Vec<bool>>,
+    skip_sync_block: bool,
 ) -> TestState {
-    let builder = TestLoopBuilder::new();
+    let mut builder = TestLoopBuilder::new();
 
-    let num_block_producer_seats = 1;
-    let num_chunk_producer_seats = num_shards;
+    let num_block_producer_seats = num_validators;
+    let num_chunk_producer_seats = num_validators;
```

Comment:
Does this make it without chunk validators only? It would be good to have some.

Reply:
Yeah this was just sort of a lazy way to get the new test to work. Added explicit num_seats params to the test with a non-producing validator in most of them